make/env.mk

```diff
@@ -35,16 +35,11 @@ ifeq ($(GOBIN),)
 GOBIN := $(GOPATH)/bin
 endif
 
-TAG := # make sure tag is never injectable as an env var
```

Comment:
I'm failing to understand why it was introduced in https://github.com/stackrox/rox/pull/6521 therefore ok, let's see how it goes without that.